app/utils/auth.py

```
@@ -118,3 +122,47 @@ async def get_current_user(
+            )
+
+        payload = auth_result.payload
+        user = UserClaims(**payload)
+        return user
+
+def get_user_authenticator_dependency() -> IUserAuthenticator:
+    return ClerkUserAuthenticator()
+
+async def get_authenticated_user(
+    request: Request,
+    auth_header: HTTPAuthorizationCredentials = Depends(http_bearer_security_schema),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)